mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +24,14 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
```

Comment:
how about creating a test for it...?

Reply:
Is it covered now?

Reply:
Yes, @bartoszmajsak. This is covered. https://github.com/arquillian/smart-testing/pull/118/files#diff-7e2181222a897b1b345a6e3cc8712122R31